diff --git a/flint-simulation.cabal b/flint-simulation.cabal
index e2309af..5dd4b44 100644
--- a/flint-simulation.cabal
+++ b/flint-simulation.cabal
@@ -45,8 +45,8 @@ executable eflint-server
                       , filepath >= 1.4.2
                       , directory >= 1.3.6
                       , exploring-interpreters >= 1.3.0.0
-                      , fgl >= 5.7 
-                      , mtl >= 2.2 
+                      , fgl >= 5.7
+                      , mtl >= 2.2
   hs-source-dirs:      src
   default-language:    Haskell2010
   ghc-options:         -fwarn-incomplete-patterns -fwarn-unused-imports
@@ -87,17 +87,18 @@ executable eflint-repl
   ghc-options:         -fwarn-incomplete-patterns -fwarn-unused-imports
 
 library
-  exposed-modules :
-  other-modules: State
-                 Spec
-                 Parse
-                 Print
+  exposed-modules : State
+                  , Spec
+                  , Parse
+                  , Print
+                  , Interpreter
+                  , NewExplorer
+  other-modules:
                  StaticEval
                  Eval
                  Saturation
                  Binders
                  JSON
-                 Interpreter
                  Util
                  Options
   build-depends:       base >=4.9
@@ -111,9 +112,9 @@ library
                       , network >= 3.1
                       , filepath >= 1.4.2
                       , directory >= 1.3.6
-                      , exploring-interpreters >= 1.3.0.0 
-                      , fgl >= 5.7 
-                      , mtl >= 2.2 
+                      , exploring-interpreters >= 1.3.0.0
+                      , fgl >= 5.7
+                      , mtl >= 2.2
   hs-source-dirs:      src
   default-language:    Haskell2010
   ghc-options:         -fwarn-incomplete-patterns -fwarn-unused-imports
